scrapers/GloryHoleSwallow.yml

```diff
@@ -0,0 +1,25 @@
+name: "GloryHoleSwallow"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://gloryholeswallow.com/newtour/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Details: //span[@class="latest_update_description"]/text()
+      Date:
+        selector: //ul/comment()[contains(.,'update_date')]
+        replace:
+          - regex: .+(?:<.+>)(.+)(?:<.+>)
+          - with: $1
+        parseDate: January 02, 2006
+      Tags:
+        Name: //span[@class="tour_update_tags"]/a//font/text()
+      Image:
+        selector: (//div[@class="back"]/center/img/@src)[1]
+        replace:
+          - regex: ^
+          - with: https://gloryholeswallow.com/newtour/
```

Comment:
you put an extra `-` to the `with:`
should be
`with: https://gloryholeswallow.com/newtour/`

Reply:
Ahhh there we go